packages/react-components/src/components/ErrorBar.tsx

```diff
@@ -49,14 +49,14 @@ export interface ErrorBarStrings {
   /**
    * User is no longer on the thread.
    *
-   * See also: {@link ErrorBarStrings.sendMessageNotInThisThread} for a more specific error.
+   * See also: {@link ErrorBarStrings.sendMessageNotInChatThread} for a more specific error.
    */
-  userNotInThisThread: string;
+  userNotInChatThread: string;
 
   /**
    * Sending message failed because user is no longer on the thread.
    */
-  sendMessageNotInThisThread: string;
+  sendMessageNotInChatThread: string;
```

Comment:
Renaming this because this is from azure review comments. Since this name is a review-approved name - maybe I will just keep this since we don't have a strong feeling about this